code/src/nuvla/job/actions/utils/resource_log_fetch.py

```diff
@@ -74,8 +74,10 @@ def get_components_logs(self) -> dict:
         components = self.get_list_components()
         lines = self.resource_log.get('lines', 200)
         log = {}
+        ## FIXME
```

Comment:
Why two hash signs for comments? SonarCloud may not pick it up as code smell.

Reply:
Changed for next commit. Indeed SonarCloud did not pick up the FIXME.